lib/ama/styles/resolver.rb

```diff
@@ -12,11 +12,18 @@ def asset_path
       end
 
       def version
-        @version ||= ama_styles_default_version
+        @version || cached_ama_styles_version || ama_styles_default_version
```

Comment:
Is the 'ama_styles_default_version' neccessary at the end? 
If I am reading this correctly 'cached_ama_styles_version' is always returning proper version or default one?

Reply:
good catch. since I'm checking for valid cache entry and returning the default version, we're covered